### PR TITLE
docs: Fix cockpit installation guide

### PR DESCRIPTION
**Pull request by sbernauer**
# Description

*Please add a description here. This will become the commit message of the merge request later.*

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes

```[tasklist]
# Author
- [ ] Changes are OpenShift compatible
- [ ] CRD changes approved
- [ ] Helm chart can be installed and deployed operator works
- [ ] Integration tests passed (for non trivial changes)
```

```[tasklist]
# Reviewer
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added
- [ ] Documentation added or updated
- [ ] Changelog updated
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
```

### CODE REVIEW

File: web/src/api/schema.d.ts [hunk 1]
```diff
@@ -157,8 +157,6 @@ export interface components {
   pathItems: never;
 }
 
-export type $defs = Record<string, never>;
-
```
Comment (fhennig) [inline web/src/api/schema.d.ts:161]: Uhm, what is going on here?

**Review by fhennig — CHANGES_REQUESTED** [review]
Just a question, but rest looks sensible

File: web/src/api/schema.d.ts (hunk 1 above)
Comment (sbernauer) [reply web/src/api/schema.d.ts:161]: I ran `cargo xtask gen-openapi` according to the README as the commit hooks yelled at me

Comment (fhennig) [reply web/src/api/schema.d.ts:161]: Ok haha

**Review by fhennig — APPROVED** [review]
ship it

**Pull request merged**